plugins/jira-dashboard-common/api-report.md

```diff
@@ -59,7 +67,7 @@ export type Project = {
     '48x48': string;
   };
   projectTypeKey: string;
-  projectCategory?: {
+  projectCategory: {
```

Comment:
This was add in issue #34.  We need to keep those changes to still support Jira Cloud

Reply:
updated